app/v1/models/response/retrieveBsas/ukProperty/Metadata.scala

```diff
@@ -35,7 +35,7 @@ object Metadata {
   implicit val reads: Reads[Metadata] = (
     (JsPath \ "inputs" \ "incomeSourceType").read[IncomeSourceType].map(_.toTypeOfBusiness) and
       JsPath.read[AccountingPeriod] and
-      (JsPath \ "metadata" \ "taxYear").read[String].map(DesTaxYear.fromDes) and
+      (JsPath \ "metadata" \ "taxYear").read[Int].map(DesTaxYear.fromDesIntToString) and
```

Comment:
as above

Reply:
we implemented this in individual_calculations. Just did the same...

Reply:
Don't merge until we update the stud responses.